### PR TITLE
chore: add build:dev commands

### PR DESCRIPTION
**Pull request by quanho**

### CODE REVIEW

File: packages/picasso.js/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
   "module": "dist/picasso.esm.js",
   "scripts": {
     "build": "cross-env NODE_ENV=production rollup --config ../../rollup.config.js",
+    "build:dev": "cross-env rollup --config ../../rollup.config.js",
```
Comment (T-Wizard) [inline packages/picasso.js/package.json:30]:
```suggestion
    "build": "rollup --config ../../rollup.config.js",
```
I would suggest removing NODE_ENV=production here. As i think it is enough to have in the top level package.json
(same for the package.json in hammer and q plugins)

File: plugins/hammer/package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
   "module": "dist/picasso-hammer.esm.js",
   "scripts": {
     "build": "cross-env NODE_ENV=production rollup --config ../../rollup.config.js",
+    "build:dev": "cross-env rollup --config ../../rollup.config.js",
```
Comment (nilzona) [inline plugins/hammer/package.json:18]: no need for `cross-env` unless you're actually setting an environment variable

File: plugins/q/package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
   "module": "dist/picasso-q.esm.js",
   "scripts": {
     "build": "cross-env NODE_ENV=production rollup --config ../../rollup.config.js",
+    "build:dev": "cross-env rollup --config ../../rollup.config.js",
```
Comment (nilzona) [inline plugins/q/package.json:18]: same here

**Comment by veinfors** [comment]
Isn't it better to make sure the `build:watch` command builds the way you want (including source maps etc.) instead of introducing yet another build script? I feel it might cause confusion around which one to use otherwise... But maybe I'm missing something.

File: packages/picasso.js/package.json (hunk 1 above)
Comment (quanho) [reply packages/picasso.js/package.json]: Good point! Updated!

File: plugins/q/package.json (hunk 3 above)
Comment (quanho) [reply plugins/q/package.json]: Updated!

File: plugins/hammer/package.json (hunk 2 above)
Comment (quanho) [reply plugins/hammer/package.json]: Updated!

**Review by T-Wizard — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production FORCE_COLOR=1 lerna run build --stream",
+    "build:dev": "FORCE_COLOR=1 lerna run build --stream",
```
Comment (T-Wizard) [inline package.json:10]: this is missing cross-env
```suggestion
    "build:dev": "cross-env FORCE_COLOR=1 lerna run build --stream",
```

Comment (quanho) [reply package.json]: Updated!

**Pull request merged**